client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -91,41 +95,43 @@ module.exports = React.createClass( {
 		}
 		return (
 			<form id="site-settings" onSubmit={ this.submitForm } onChange={ this.markChanged }>
-				<fieldset>
-					<label htmlFor="wgaCode">{ this.translate( 'Google Analytics Tracking ID', { context: 'site setting' } ) }</label>
-					<input
+				<FormFieldset>
+					<FormLabel htmlFor="wgaCode">
+						{ this.translate( 'Google Analytics Tracking ID', { context: 'site setting' } ) }
+					</FormLabel>
+					<FormTextInput
 						name="wgaCode"
 						id="wgaCode"
-						type="text"
 						value={ this.state.wga.code }
-						onChange={this.handleCodeChange}
+						onChange={ this.handleCodeChange }
 						placeholder={ placeholderText }
 						disabled={ this.state.fetchingSettings }
 						onClick={ this.recordEvent.bind( this, 'Clicked Analytics Key Field' ) }
-						onKeyPress={ this.recordEventOnce.bind( this, 'typedAnalyticsKey', 'Typed In Analytics Key Field' ) }
-					/>
-					<p className="settings-explanation"><a href="https://support.google.com/analytics/answer/1032385?hl=en" target="_blank">{
-						this.translate( 'Where can I find my Tracking ID?' )
+						onKeyPress={ this.recordEventOnce.bind( this, 'typedAnalyticsKey', 'Typed In Analytics Key Field' ) } />
+					<FormSettingExplanation>
+						<a href="https://support.google.com/analytics/answer/1032385?hl=en" target="_blank">
+							{ this.translate( 'Where can I find my Tracking ID?' ) }
+						</a>
+					</FormSettingExplanation>
```

Comment:
I'm not sure how often this component is used, but right now it just looks like a heavy wrapper for `<p />`. On the other hand, it has a name that connotes quite a bit of semantic purpose. I would personally rather see the information given as props instead of sent as children.

```js
<FormSettingExplanation
	link="https://support.google.com/analytics/answer/103285?hl=en"
	explanation={ this.translate( 'Where can I find my Tracking ID?' ) } />
```

Of course, this would require some modification to that component, but I don't see the value in abstracting `<p />`